src/compass/utils/lut.py

```diff
@@ -103,18 +114,246 @@ def compute_geocoding_correction_luts(burst, dem_path,
         in seconds as the function of the azimuth time and slant range.
         This correction needs to be added to the SLC tagged azimuth time to
         get the corrected azimuth times.
+
+    [rg_set, az_set]: list, np.ndarray
+        List of numpy.ndarray containing SET in slant range and azimuth directions
+        in meters. These corrections need to be added to the slC tagged azimuth
+        and slant range times.
     '''
+    # Get DEM raster
```

Comment:
```suggestion
    if not os.path.isfile(dem_path):
        raise FileNotFoundError(f'Cannot find the dem file: {dem_path}')
    # Get DEM raster
```
Check `dem_path` before raster init.